src/tray/templates/services/app/secret.yml.j2

```diff
@@ -7,6 +7,17 @@ metadata:
   name: "{{ ralph_secret_name }}"
   namespace: "{{ project_name }}"
 data:
+  # Swift
+  RALPH_SWIFT_OS_AUTH_URL: "{{ RALPH_VAULT.RALPH_SWIFT_OS_AUTH_URL | b64encode }}"
```

Comment:
All variable are mandatory ?

Reply:
To connect to swift yes. Maybe we should introduce more flexibility in backends configuration and allow to make them optional. Is this what you have in mind?

Reply:
If they are mandartory, they should not be commented in the main file. WDYT ?

Reply:
They are mandatory only if you use this backend.

Reply:
Sorry you mean mandatory from a jinja/ansible perspective, not Ralph! You are perfectly right, this is not consistent.

Reply:
This is now fixed in a way simpler way. Thanks @lunika :muscle:

Reply:
![](https://media.giphy.com/media/3o7TKrbXUnCJkzAWu4/giphy.gif)